azure-pipelines-ui-tests.dfe.yml

```diff
@@ -0,0 +1,212 @@
+
+schedules:
+- cron: 0 5 * * 1,2,3,4,5
```

Comment:
Since we're getting rid of the retry on failure tasks I would suggest we add another schedule for 7AM